core/math/vector2.h

```diff
@@ -32,8 +32,8 @@
 #define VECTOR2_H
 
 #include "core/math/math_funcs.h"
-#include "core/string/ustring.h"
```

Comment:
Moving this to the `.cpp` is pretty much why I had to add some `#include "core/templates/vector.h"` and `#include "core/os/memory.h"` in various unrelated files... It was a bit far fetched to depend on String being included in Vector2 to get access to `memnew_placement` and `Vector` IMO ;)